examples/loki-stack/role-hack.yaml

```
@@ -0,0 +1,31 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: loki-netobserv-tenant
+rules:
+- apiGroups:
+  - 'loki.grafana.com'
+  resources:
+  - infrastructure
+  resourceNames:
+  - logs
+  verbs:
+  - 'get'
+  - 'create'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: loki-netobserv-tenant
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: loki-netobserv-tenant
+subjects:
+- kind: ServiceAccount
+  name: flowlogs-pipeline
+  namespace: netobserv
+- kind: ServiceAccount
+  name: network-observability-plugin
```

this guy has been renamed
```suggestion
  name: netobserv-plugin
```